cmd/juju/caas/add.go

```diff
@@ -644,7 +647,15 @@ func (c *AddCAASCommand) Run(ctx *cmd.Context) (err error) {
 	if clusterName != "" {
 		clusterName = fmt.Sprintf("%q ", clusterName)
 	}
-	successMsg := fmt.Sprintf("k8s substrate %sadded as cloud %q%s", clusterName, c.caasName, storageMsg)
+	storageMsg := "with no configured storage provisioning capability"
+	if !c.skipStorage {
+		storageMsg = fmt.Sprintf(`with storage provisioned
+by the existing %q storage class
+and operator storage provisioned by the workload storage class`,
+			newCloud.Config[k8sconstants.WorkloadStorageKey])
```

Comment:
What about the case where operator and workload storage are provisioned by different storage classes